### PR TITLE
Home page button

### PR DESCRIPTION
**Pull request by Pawel1Golabek**
Dobre repo? ^^

### CODE REVIEW

File: src/app/shared/components/header/header.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <header class='header'>
   <div class="header-icon__wrapper">
-    <div class="header-icon__wrapper__icon"></div>
-    <a class="header-icon__wrapper__link" href="#welcome">Home</a>
+    <a href = "#welcome" id = "headerIconLink" ><div class="header-icon__wrapper__icon" id = "headerIcon"></div></a>
```
Comment (joannakrasucka) [inline src/app/shared/components/header/header.component.html]: Please merge those anchors into one :)

File: src/app/shared/components/header/header.component.scss [hunk 2]
```diff
@@ -22,7 +23,21 @@
...
+      cursor: default;
     }
+
+    &__reactorIcon {
+      width: 25px;
+      height: 22.569px;
+      display: inline-block;
+      background-image: url("../../../../assets/icons/arrow-left.svg");
+      margin-right: 1rem;
+      fill: var(--Ocean-Blue, #2071B5);
```
Comment (joannakrasucka) [inline src/app/shared/components/header/header.component.scss]: Please use project variables

File: src/app/shared/components/header/header.component.scss [hunk 3]
```diff
@@ -22,7 +23,21 @@
       display: inline-block;
       background-image: url("../../../../assets/icons/home.svg");
       margin-right: 1rem;
+      cursor: default;
     }
+
+    &__reactorIcon {
+      width: 25px;
```
Comment (joannakrasucka) [inline src/app/shared/components/header/header.component.scss]: Pixels?

File: src/app/shared/components/header/header.component.ts [hunk 4]
```diff
@@ -7,5 +9,31 @@ import { Component } from '@angular/core';
   styleUrl: './header.component.scss'
 })
 export class HeaderComponent {
+  ngOnInit() {
+    let location:string = window.location.href;
+    if(location.includes("reactors")){
+      var htmlElement;
```
Comment (joannakrasucka) [inline src/app/shared/components/header/header.component.ts]: Please replace var with let

**Review by Pawel1Golabek — COMMENTED** [review]
Fixed issues from prev code review

**Comment by Pawel1Golabek** [comment]
fixed :)

**Review by joannakrasucka — APPROVED** [review]
Great work!

**Pull request merged**